package.yaml

```diff
@@ -1,4 +1,17 @@
-name: list-zipper
+name: nonempty-zipper
+version: 1.0.0.0
+github: "freckle/nonempty-zipper"
+license: MIT
+author: "Freckle R&D"
+maintainer: "engineering@freckle.com"
+copyright: "2021 Freckle Education"
```

Comment:
We copyright things to "Renaissance Learning Inc", IIUC.

Reply:
Done!